b2sdk/stream/base.py

```diff
@@ -0,0 +1,9 @@
+import io
+
+
+class ReadOnlyMixin(object):
+    def writeable(self):
+        return False
+
+    def write(self, data):
+        raise io.UnsupportedOperation('Stream with hash cannot be written')
```

Comment:
but really
```suggestion
        raise io.UnsupportedOperation('Cannot accept a write to a read-only stream')
```
because someone might one day use this mixing for some other file-like object and then the exception message would not make any sense

Reply:
Thanks for a catch - I really missed it - but this fix with `too` - you know - it was in the code for probably 4 years - maybe you should read all the code for audit and fix such things - so we would not be fixing stupid stuff on really hard review and your mind would be less distracted :) 

but this suggestions - thx - it is just that I copied from StreamWithHash to mixin and forgot to change the string

Reply:
and because you pointed out `too` now I can commit this great suggestion :(

Reply:
I see your frustration with github is increasing over time ;)